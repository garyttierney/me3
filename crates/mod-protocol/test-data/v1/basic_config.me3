profileVersion = "v1"

[[packages]]
id = "my-mod"
source = "./mod"

[[natives]]
path = "my_native.dll"
optional = true
