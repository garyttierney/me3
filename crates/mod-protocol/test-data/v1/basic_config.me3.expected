V1(
    ModProfileV1 {
        supports: [],
        natives: [
            Native {
                inner: ModFile {
                    name: "my_native",
                    path: "my_native.dll",
                    enabled: true,
                    optional: true,
                },
                initializer: None,
                load_before: [],
                load_after: [],
            },
        ],
        packages: [
            Package {
                inner: ModFile {
                    name: "my-mod",
                    path: "./mod",
                    enabled: true,
                    optional: false,
                },
                load_before: [],
                load_after: [],
            },
        ],
        savefile: None,
        start_online: None,
        disable_arxan: None,
    },
)
